docs/core/sdk/file-based-apps.md

```diff
@@ -147,7 +152,7 @@ Convert your file-based app to a traditional project by using the `dotnet projec
 dotnet project convert file.cs
 ```
 
-This command creates a `.csproj` file with equivalent SDK and properties. All `#` directives are removed from the `.cs` file and turned into elements in the corresponding `.csproj` file.
+This command creates a `.csproj` file with equivalent SDK and properties. The command removes all `#:` directives from the `.cs` file and turns them into elements in the corresponding `.csproj` file.
```

Comment:
Would be good to make it clear the original .cs file is left untouched, and that the created project file is placed in a directory next to the original .cs file, along with a copy of the original .cs file with the `#:` directives removed.